java/src/main/java/com/google/oak/client/BUILD

```diff
@@ -33,3 +34,17 @@ java_library(
         "//java/src/main/java/com/google/oak/util",
     ],
 )
+
+# Android-friendly flavor of OakClient.
+android_library(
```

Comment:
Do we have dependencies to build Android libraries in Nix?

cc @tiziano88

Reply:
mitigated by removing the android_library